README.md

```diff
@@ -39,79 +41,20 @@ Each package has its own **node_modules** directory that contains symbolic links
 
 > See [supported platforms](./docs/learning/SupportedPlatforms.md) for further information.
 
-## Build Instructions
 
-1. Clone repository (first time) with `git clone` or pull updates to the repository (subsequent times) with `git pull`
+## Contribution
+
+### Developer Quick Start
+
+The following instructions will quickly set the repo up for you to edit the source code and contribute:
+
+1. Clone the repository locally
```

Comment:
for consistency, thoughts on adding the actual command?

Reply:
yep, added